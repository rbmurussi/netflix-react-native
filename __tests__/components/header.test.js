import React from "react";
import renderer from "react-test-renderer";
import Header from "../../components/Header";

describe.only("renders correctly", () => {
  const JSON = renderer.create(<Header />).toJSON();
  expect(JSON).toMatchSnapshot();
});

describe("Header", () => {
  test("BIG", () => {
    expect(1).toBe(1);
  });
  test("SMALL", () => {
    expect(1).toBe(1);
  });
  test("MEDIUM", () => {
    expect(1).toBe(1);
  });
});
