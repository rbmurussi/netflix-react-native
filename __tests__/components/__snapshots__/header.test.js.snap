// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingBottom": 0,
        "paddingLeft": 25,
        "paddingRight": 25,
        "paddingTop": 25,
        "width": "100%",
      },
    ]
  }
>
  <Image
    resizeMode="contain"
    source={
      Object {
        "testUri": "../../../assets/logo.png",
      }
    }
    style={
      Array [
        Object {
          "height": 40,
          "width": 20,
        },
      ]
    }
  />
  <View
    accessible={true}
    focusable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#fff",
            "fontSize": 18,
            "letterSpacing": 0.1,
          },
        ]
      }
    >
      SÃ©ries
    </Text>
  </View>
  <View
    accessible={true}
    focusable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#fff",
            "fontSize": 18,
            "letterSpacing": 0.1,
          },
        ]
      }
    >
      Filmes
    </Text>
  </View>
  <View
    accessible={true}
    focusable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#fff",
            "fontSize": 18,
            "letterSpacing": 0.1,
          },
        ]
      }
    >
      Minha lista
    </Text>
  </View>
</View>
`;
